subsys/bootloader/bl_client/bl_client.c

```diff
@@ -59,6 +59,20 @@ static const struct bl_secp256r1_abi *get_bl_secp256r1_abi(void)
 	return (const struct bl_secp256r1_abi *)abi;
 }
 
+static const struct bl_validate_fw_abi *get_bl_validate_firmware_abi(void)
+{
+	static const struct fw_abi_info *abi;
+
+	if (!abi) {
+		abi = fw_abi_find(BL_VALIDATE_FW_ABI_ID,
+				CONFIG_BL_VALIDATE_FW_ABI_FLAGS,
```

Comment:
Are you missing the patch which adds these kconfigs? I don't see them defined in this PR.

Reply:
I get a compile error because of these missing.

Reply:
I'm pretty sure they are located in the FW_METADATA template PR but i'm not 100% certain.